Sources/App/SlackCommand+Options.swift

```diff
@@ -6,13 +6,17 @@ extension SlackCommand {
     enum Option {
         static let branch = "branch"
         static let repo = "repo"
+        static let version = "version"
     }
+}
 
-    static func branch(fromOptions options: [String]) -> String? {
-        let branchOptionPrefix = Option.branch + ":"
-        let branch = options.dropFirst()
-            .first { $0.hasPrefix(branchOptionPrefix) }?
-            .dropFirst(branchOptionPrefix.count)
+extension SlackCommandMetadata {
+    func value(forOption option: String) -> String? {
+        let components = text.components(separatedBy: " ")
```

Comment:
Starting to think that we shouldn't simply split the `text` into `components` directly in the `init` of `SlackCommandMetaData` so we can access `components` directly for every command, instead of writing that `text.components(separatedBy: " ")` in lots of places in the code and splitting multiple times for nothing.

(and btw `text.split(separator: " ")` might be better than `components(separatedBy:)` to avoid the round trip to Foundation or the extra allocations?)

Reply:
👍